### PR TITLE
Update test app logging service and instruction text

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR updates the Edge test app to use the Core 2.0 logging service and some instruction/comment text

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/app/src/main/java/com/adobe/marketing/tester/TestApplication.java [hunk 1]
```diff
@@ -12,20 +12,21 @@
...
 import com.adobe.marketing.mobile.MobileCore;
 import com.adobe.marketing.mobile.edge.consent.Consent;
 import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.services.Log;
 import java.util.Arrays;
 
 public class TestApplication extends Application {
 
-	private static final String LOG_TAG = "TestApplication";
+	private static final String LOG_TAG = "EdgeTestApplication";
```
Comment (kevinlind) [inline code/app/src/main/java/com/adobe/marketing/tester/TestApplication.java:26]: MainActivity is using `LOG_TAG = "TestApp"`. Can you consolidate the two? Maybe rename to "EdgeTestApp", make `public static final` and then static import this LOG_TAG into MainActivity?

**Review by emdobrin — APPROVED** [review]

File: code/app/src/main/java/com/adobe/marketing/tester/TestApplication.java [hunk 2]
```diff
@@ -36,10 +37,10 @@ public void onCreate() {
 		MobileCore.setLogLevel(LoggingMode.VERBOSE);
 		MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
 
-		// register Adobe extensions
+		// Register Adobe extensions
 		MobileCore.registerExtensions(
 			Arrays.asList(Edge.EXTENSION, Identity.EXTENSION, Consent.EXTENSION, Assurance.EXTENSION),
-			o -> Log.d(LOG_TAG, "Mobile SDK was initialized")
+			o -> Log.debug(LOG_TAG, LOG_SOURCE, "Mobile SDK was initialized")
```
Comment (addb) [inline code/app/src/main/java/com/adobe/marketing/tester/TestApplication.java:43]: nit: Should we reword it to something like `AEP Mobile SDK initialized`

**Review by addb — APPROVED** [review]
